.github/workflows/build-image.yml

```diff
@@ -24,6 +24,9 @@ jobs:
     strategy:
```

Comment:
Why not add CUDA 12.0 directly?

Reply:
there's no cuda12.0 image yet, using the [latest one](https://docs.nvidia.com/deeplearning/frameworks/pytorch-release-notes/rel-22-12.html#rel-22-12) which is 11.8